README.md

```diff
@@ -0,0 +1,22 @@
+# World Portals
+
+A module to add teleportation portals. Portals currently occupy a block and transport any entities that enter the occupied block to the portals destination world position.
+
+## Future Features
```

Comment:
Unless there is good reason, I would push this down to the very end.
(keep in mind that these are ToDos, that may or may not ever get implemented. This readme is for the players as much as is it for the developers, and both of them are concerned primarily with what the module *currently* offers, not what it *might* offer some day)